tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/NativeMobile/NativeMobileAdditionalInformation/Supplier Get NativeMobileAdditionalInformation.feature

```diff
@@ -25,17 +24,11 @@ Scenario: Native Mobile Additional Information are retreived for the solution
     And the string value of element additional-information is Some more info
 
 @3610
-Scenario: Native Mobile Additional Information are retrieved for the solution where no solutiondetail exists
+Scenario: Native Mobile Additional Information are retrieved empty for the solution
```

Comment:
Suggestion: `native mobile additional information is not retrieved when no additional information exists`

Reply:
Updated Scenario description to `native mobile additional information is not retrieved when no additional information exists`